deploy/crds/aerogear_v1alpha1_unifiedpushserver_crd.yaml

```diff
@@ -9,8 +9,6 @@ spec:
     listKind: UnifiedPushServerList
     plural: unifiedpushservers
     singular: unifiedpushserver
-    shortNames:
-      - ups
```

Comment:
Are short names not good? I prefer `oc get ups` rather than `oc get unifiedpushserver`

Reply:
wtf? I didn't delete that one :S

`operator-sdk generate openapi` deleted it probably. gonna have a look.